sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/AbstractResourceMapper.java

```diff
@@ -0,0 +1,90 @@
+package com.mesosphere.sdk.offer.evaluate;
+
+import com.mesosphere.sdk.offer.LoggingUtils;
+import com.mesosphere.sdk.offer.ResourceUtils;
+import com.mesosphere.sdk.specification.ResourceSpec;
+import com.mesosphere.sdk.specification.VolumeSpec;
+import org.apache.mesos.Protos;
+import org.slf4j.Logger;
+
+import java.util.Collection;
+import java.util.Optional;
+
+abstract class AbstractResourceMapper {
+    final Logger logger;
+    final Optional<String> resourceNamespace;
+
+    AbstractResourceMapper(Optional<String> resourceNamespace) {
+        logger = LoggingUtils.getLogger(getClass(), resourceNamespace);
+        this.resourceNamespace = resourceNamespace;
+    }
+
+    Optional<ResourceLabels> findMatchingDiskSpec(
```

Comment:
FWIW, so far most of the code goes with this style:
```
void func(A a, B b) {
```
wrapping to this if the above doesn't fit 120col:
```
void func(
        A a, B b) {
```
then finally this if that doesn't fit either:
```
void func(
        A a,
        B b) {
```
...but that said, I'm open to just picking a style and enforcing it via lint at some point. There hasn't really been an 'official style' yet.

Reply:
Sorry, that was the scala-ey me. However I propose this : 

we always do this : 
```
void func(A a, B b) {
```
and if that doesn't fit in to 120C then we _always_ adhere to : 
```
void func(
        A a,
        B b
) {
```
I feel like this would be the most simplest and readable for someone new to codebase.
I volunteer to add this styling rule and reformat the codebase to stick to the rule.

Reply:
That sounds fine, I'd subjectively prefer this style:

```
void func(
        A a,
        B b) {
    ... code here ...
}
```

...but I could be convinced. This would be a separate PR in any case. Maybe we could do a poll...